### PR TITLE
Feature/minor bugs UI

### PR DESCRIPTION
**Pull request by Kaizen-7**

### CODE REVIEW

File: buildSrc/src/main/java/AppConfig.kt [hunk 1]
```diff
@@ -24,7 +24,7 @@ import org.gradle.api.JavaVersion
 
 object Config {
     const val minSdk = 24
-    const val compileSdk = 29
+    const val compileSdk = 30
```
Comment (astagi) [inline buildSrc/src/main/java/AppConfig.kt:27]: @Kaizen-7 why?

**Review by astagi — CHANGES_REQUESTED** [review]

File: buildSrc/src/main/java/AppConfig.kt (hunk 1 above)
Comment (Kaizen-7) [reply buildSrc/src/main/java/AppConfig.kt:27]: The implementation of the new jetpack startup library which provides "initializer" class requires this version of the compile SDK in order to work

Comment (astagi) [reply buildSrc/src/main/java/AppConfig.kt:27]: @Kaizen-7 I know that compileSdkVersion value is used by Gradle to compile against new APIs, but how jetpack startup library affects older device? does it work / require a fallback for devices with AndroidSDK < 30?

Comment (Kaizen-7) [reply buildSrc/src/main/java/AppConfig.kt:27]: yes it works on older device and APIs, so it doesn't affect minSdkVersion at all

Comment (astagi) [reply buildSrc/src/main/java/AppConfig.kt:27]: Great! Thank you @Kaizen-7

Comment (rawmain) [reply buildSrc/src/main/java/AppConfig.kt:27]: > yes it works on older device and APIs, so it doesn't affect minSdkVersion at all

Confirmed. No issues on real-devices down to API 24 = OK even with further minSdk downgrade to API 23 (_by applying PR #113 commits to current develop branch code_).

**Pull request merged**